tests/excerptexport/test_downloads.py

```diff
@@ -24,7 +24,8 @@ def test_file_download(self, *args):
         excerpt = Excerpt.objects.create(name='Neverland', is_active=True, is_public=True, owner=user,
                                          bounding_geometry=bg)
         extraction_order = ExtractionOrder.objects.create(excerpt=excerpt, orderer=user)
-        output_file = OutputFile.objects.create(mime_type='test/plain', extraction_order=extraction_order,
+        export = extraction_order.exports.create(file_format='fgdb')
+        output_file = OutputFile.objects.create(mime_type='test/plain', export=export,
```

Comment:
I wonder though whether we should persist the MIME type at all, seeing that we only put hard-coded values into that field. We could just as well hard-code the value in the view that delivers the file and currently uses the field. Or, if we want to be more flexible, determine the MIME type dynamically (either when persisting the file (and store it in the existing field), or when delivering it) either from the filename extension using the [`mimetypes`](https://docs.python.org/3/library/mimetypes.html) module or from the file content using `/usr/bin/file --mime` (which uses libmagic) or any of the several [python wrappers of libmagic](http://stackoverflow.com/a/12297929/674064).